src/perform_step/low_order.jl

```diff
@@ -269,7 +327,7 @@ end
   for j = 1:length(dW)
     @.. Kj = K + sqdt*@view(L[:,j]) # This works too
```

Comment:
it wants this to be `@.`? Seems a bit odd.

Reply:
I think it will need an `if is_diagonal_noise(integrator.sol.prob)` branch, similar to the one from `RKMilGeneral()`. I think the error essentially is due to the matrix-like form of u0.

Reply:
Oh yes. `@..` specializes in a way that requires a 1-1 map.